lib/liquid/combinators/tags/for.ex

```diff
@@ -51,6 +51,22 @@ defmodule Liquid.Combinators.Tags.For do
   alias Liquid.Combinators.{General, Tag}
   alias Liquid.Combinators.Tags.Generic
 
+  @type t :: [
+          for: [
+            for_statements: [
+              variable: String.t(),
+              value: LexicalToken.value(),
+              for_params: [
+                [offset: Integer.t() | String.t()]
+                | [limit: Integer.t() | String.t()]
+              ],
+              for_body:
+                Liquid.t()
+                | [else: Generic.else_tag()]
```

Comment:
Must be `Liquid.t() | Generic.else_tag()` because `[else: Generic.else_tag()]` == `[else: [else: [String.t() | Liquid.t()]]]`